fitwsarah_frontend/src/components/footer/footerLoggedIn/footerLoggedIn.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './footerLoggedIn.css'; 
+
+function FooterLoggedIn() {
+  return (
+    <div className="footer-logged-in">
+      {/* Content goes here */}
+    </div>
+  );
+}
+export default FooterLoggedIn;
```

Comment:
Did you forget to add the footer?

Reply:
No, I simply haven't made the logged-in footer so far I just added it in the files to make sure that when we actually make it we don't waste time configuring it..